jenkins/scripts/utils.sh

```diff
@@ -60,15 +60,11 @@ vm_healthy()
     KEY="${2:?}"
     SERVER="${3:?}"
 
-    cloud_init_status=$(ssh -o ConnectTimeout=2 -o StrictHostKeyChecking=no \
+    if ssh -o ConnectTimeout=2 -o StrictHostKeyChecking=no \
         -o UserKnownHostsFile=/dev/null -i "${KEY}" \
-        "${USER}"@"${SERVER}" cloud-init status --long --wait)
-    if echo "${cloud_init_status}" | grep "error"; then
-        echo "There was a cloud-init error:"
-        echo "${cloud_init_status}"
-        return 1
```

Comment:
Not every word "error" means fatal. Instead, if the cloud-init exits with error, it is considered fatal.

Reply:
No I just simplified it so that we run it directly in the `if`. When it returns with exit code 0 that counts as `if true`, when it returns something else it counts as `if false`.
It will also output directly everything so we can see it when it runs.

Reply:
you mean even if cloud_init_status has errors word, checking the status code itself is enough?

Reply:
All this applies of course only if the cloud-init script contains `set -e` or is run in a way that it can fail. Otherwise it will just return last commands return value, which would be misleading.